package.json

```diff
@@ -30,7 +30,7 @@
     "angular-animate": "1.5.11",
     "angular-drag-and-drop-lists": "2.0.0",
     "angular-moment": "1.0.0",
-    "angular-patternfly": ">=4.11.8 <5.0.0",
+    "angular-patternfly": ">=4.18.0 <5.0.0",
```

Comment:
Looks like Console is at "angular-patternfly": "4.14.9":  https://github.com/openshift/origin-web-console/blob/master/bower.json#L16

1) I think Catalog bower.json needs to be updated  to "angular-patternfly": ">=4.18.0 <5.0.0",
2) Assuming a Console PR next to update Console bower.json to:  "angular-patternfly": "4.18.0" from  currently "angular-patternfly": "4.14.9"

Reply:
Added bower.json.
Yes, when console is upgraded to a version including this change it will also need to upgrade to angular-patternfly 4.18.0 or higher.